src/BinSkim.Rules/PERules/BA4001.ReportPortableExecutableCompilerData.cs

```diff
@@ -15,16 +15,14 @@
 namespace Microsoft.CodeAnalysis.IL.Rules
 {
     [Export(typeof(Skimmer<BinaryAnalyzerContext>)), Export(typeof(ReportingDescriptor)), Export(typeof(IOptionsProvider))]
-    public class ReportPECompilerData : WindowsBinaryAndPdbSkimmerBase, IOptionsProvider
+    public class ReportPortableExecutableCompilerData : PEBinarySkimmerBase, IOptionsProvider
     {
         /// <summary>
         /// BA4001. This reporting rule writes compiler data to AppInsights and 
         /// a CSV file (if configured) for every compilation unit that's scanned.
         /// </summary>
         public override string Id => RuleIds.ReportPortableExecutableCompilerData;
 
-        public override bool LogPdbLoadException => false;
```

Comment:
>LogPdbLoadException [](http://example.com/codeflow?start=29&length=19)

the PEBinarySkimmerBase does not log anything in case we don't find a PDB. Also, this property is only available for the WindowsBinaryAndPdbSkimmerBase.

Reply:
Also, with this change, we will try to analyze any PE available. If we find the PDB, great! If not, it will just skip.